polkadot/node/subsystem-types/src/messages.rs

```diff
@@ -781,6 +781,9 @@ pub enum RuntimeApiRequest {
 	/// Get the maximum uncompressed code size.
 	/// `V12`
 	ValidationCodeBombLimit(SessionIndex, RuntimeApiSender<u32>),
+	/// Get the paraids at the relay parent.
+	/// `V14`
+	ParaIds(SessionIndex, RuntimeApiSender<Vec<ParaId>>),
```

Comment:
@alindima did you mean for me to put the SessionIndex here in the API, or just part of `polkadot/node/core/runtime-api/src/cache.rs` ?